index.html

```diff
@@ -227,5 +234,28 @@ <h1>Skills</h1>
             </div>
         </div>
     </section>
+
+    <footer class="contact">
+        <div>
+            <h5>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h5>
+            <br>
+            <form>
```

Comment:
- [ ] Form uses the `action` and `method` attributes to integrate with [formspree service](https://formspree.io/html).
- [ ] Kindly add `method` and also use `action` attributes.
- [ ] `action` value should be similar to `https://formspree.io/f/{form_id}`
- [ ] `method` value equals `POST`

**Check below my suggestions:**
```suggestion
            <form method='POST' action=' https://formspree.io/f/{form_id}'>
```

- [ ] Please go to [formspree service](https://formspree.io/html) and follow the instruction. You will get one `form_id` from this website and then you just change this `id` with `{form_id}` like that `https://formspree.io/f/mzbolesd`.